interp/stdarg.ml

```diff
@@ -42,6 +42,8 @@ let wit_var =
 
 let wit_ref = make0 "ref"
 
+let wit_smart_global = make0 "smart_global"
```

Comment:
```suggestion
let wit_smart_global = make0 ~dyn:(val_tag (topwit wit_ref)) "smart_global"
```

Reply:
Happy to commit this (thanks), but what does this do?  (Or where's the doc?)